website/content/en/docs/config/vmtype/_index.md

```diff
@@ -22,13 +22,16 @@ flowchart
   intel_on_arm --  "No" --> vz["VZ"]
 ```
 
+The default vmType is QEMU in Lima prior to v1.0.
+Starting with Lima v1.0, Lima will use VZ by default on macOS (>= 13.5) for new instances,
+unless the config is incompatible with VZ.
```

Comment:
I think it would be good to give at least an example or two of configs that are incompatible, so the reader knows what you mean.

Reply:
done